React/Views/RCTRefreshControl.m

```diff
@@ -18,12 +18,17 @@ @implementation RCTRefreshControl {
   UIColor *_titleColor;
 }
 
+static unsigned short defaultHeight;
```

Comment:
I purposely made it static so that the value (which is the same for all instances of refreshControl) is only stored once. Having an instance variable means more memory consumption which in this case is not necessary. Are you sure to change it to per-instance var?